.github/workflows/kernel-checks.yml

```diff
@@ -32,15 +30,40 @@ jobs:
 
       # Collect all affected files
       - name: Collecting changed files
-        uses: lots0logs/gh-action-get-changed-files@2.1.4
+        uses: lots0logs/gh-action-get-changed-files@2.2.2
```

Comment:
I'm personally not opposed to using third party actions... but it does pose a security concern as the third party could modify their code to do nefarious things in theory - esp since you pass the github token.

Reply:
> I'm personally not opposed to using third party actions.
This already exists here. It's a version update since the current action version is soon to be deprecated
[Link](https://github.com/FreeRTOS/FreeRTOS-Kernel/actions/runs/6265515088)
```
<html><body>
<!--StartFragment-->

FreeRTOS Kernel Header Checks                                                                                                                           The following actions uses node12 which  is deprecated and will be forced to run on node16:  actions/setup-python@v2, lots0logs/gh-action-get-changed-files@2.1.4.  For more info:  https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/
--
 


                      <a class="d-block no-underline Link--secondary color-fg-default" href="https://github.com/FreeRTOS/FreeRTOS-Kernel/actions/runs/6265515088/job/17014521341#step:5:13">
                          <strong>FreeRTOS Kernel Header Checks</strong>
                        
                      </a>
                      <div class="color-fg-muted">
                        <div data-target="annotation-message.annotationContainer" class="text-small text-mono annotation--contracted">
                          <div>The `set-output` command is deprecated 
and will be disabled soon. Please upgrade to using Environment Files. 
For more information see: 
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/</div>
                        </div></div><!--EndFragment-->
</body>
</html>
```
>  but it does pose a security concern as the third party could modify their code to do nefarious things in theory
This is a tagged commit.

> esp since you pass the github token.
Any PR from a fork is granted read only permissions

Reply:
>This already exists here. It's a version update since the current action version is soon to be deprecated

I know it is already used - I can see the commit change. I'm still pointing it out. 

>This is a tagged commit.

Just because its tagged doesn't mean anything - an owner of a repository could just delete the tag and set it to a new commit...  

>Any PR from a fork is granted read only permissions on the token

Ok, but would if someone manually triggered it... suddenly third party action has an owner token.